stdlib/result.gr

```diff
@@ -92,30 +204,30 @@ export let peekErr = (fn, r) => {
  *
  * @param msg: The message to prepend if the result contains an `Err`
  * @param result: The result to extract a value from
- * @returns The value inside an `Ok` result
+ * @returns The unwrapped `ok` value if the result is of the `Ok` variant, otherwise throws `msg: err`
```

Comment:
```suggestion
 * @returns The unwrapped `Ok` value if the result is the `Ok` variant, fails with the provided message otherwise
```

Reply:
```suggestion
 * @returns The unwrapped value if the Result is the `Ok` variant
```
We shouldn't document throws in a return value. We have an issue for documenting throws with graindoc, but if you want to currently document the failure case, it should be in the description.